src/SFA.DAS.SharedOuterApi/Infrastructure/HttpRequestContentException.cs

```diff
@@ -7,6 +7,11 @@ public class HttpRequestContentException : HttpRequestException
     {
         public string ErrorContent { get; set; }
         public HttpStatusCode StatusCode { get; set; }
+
+        public HttpRequestContentException(string message, HttpStatusCode statusCode) : this(message, statusCode, "")
+        {
+        }
+
```

Comment:
This is simply adding a convenient constructor that doesn't require a message.  Most of the places we are constructing this exception is passing in an empty string for the message anyway, so this is really only moving where the empty string is set.

Reply:
One tiny comment, I don't think we should create this exception without the content string. If we do we are just creating a 
HttpRequestException() or the equivalent

Reply:
Hmm that's what we've been doing in almost all uses of it.  `HttpRequestContentException` also has the `StatusCode` that isn't in `HttpRequestException`.  Do you think that's enough to justify the empty message?  Otherwise we'll have to change the Training Provider and Commitments API calls.

Reply:
Good point